DETAILS.md

```diff
@@ -43,7 +41,9 @@ The extension installs the following tasks:
   and user password for a [Google Compute Engine][ComputeEngine] virtual machine
   running windows.
   - **[Install Google Cloud SDK][install-cloud-sdk]**: Finds or installs the
-  [Google Cloud SDK][CloudSdk].
+  [Google Cloud SDK][CloudSdk] on windows.
```

Comment:
Maybe "Windows only" at the start of the line?

Reply:
That is better. Thanks.